unblob/handlers/filesystem/romfs.py

```diff
@@ -257,41 +253,41 @@ def walk_dir(self, addr: int, parent: Optional[FileHeader] = None):
             self.inodes[addr] = file_header
         return file_header.next_filehdr
 
-    def create_symlink(self, extract_root: Path, output_path: Path, inode: FileHeader):
+    def create_symlink(self, output_path: Path, inode: FileHeader):
         target = inode.content.decode("utf-8").lstrip("/")
 
         if target.startswith(".."):
-            target_path = extract_root.joinpath(output_path.parent, target).resolve()
+            target_path = output_path.parent / target
         else:
-            target_path = extract_root.joinpath(target).resolve()
+            target_path = Path(target)
 
         # we create relative paths to make the output directory portable
         target_path = Path(os.path.relpath(target_path, start=output_path.parent))
         self.fs.create_symlink(src=target_path, dst=output_path)
 
-    def create_hardlink(self, extract_root: Path, output_path: Path, inode: FileHeader):
+    def create_hardlink(self, output_path: Path, inode: FileHeader):
         if inode.spec_info in self.inodes:
             target = str(self.inodes[inode.spec_info].path).lstrip("/")
```

Comment:
I guess this will be handled by `chop_root` later on so the `lstrip` is no longer required.

Reply:
Agreed, and removing this even conversion to `str` and back to `Path` can be avoided.